app/src/lib/source-map-support.ts

```diff
@@ -39,9 +38,9 @@ function retrieveSourceMap(source: string) {
         return { url: Path.basename(path), map: xhr.responseText }
       }
     } catch (error) {
-      return
+      return null
```

Comment:
This is enforced by the TS type definitions of `source-map-support` (with the `require` statement, `source-map-support` was typed as `any`).

Based on the module internal logic, it should change things when returning here `null` instead of `undefined`, since there's a loose check ([source](https://github.com/evanw/node-source-map-support/blob/v0.4.15/source-map-support.js#L168)), so this change should be safe.